packages/Skeleton/withSkeleton.jsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { SkeletonContext } from './SkeletonProvider'
+
+class SkeletonRenderer extends React.Component {
```

Comment:
Would it make sense to make this `SFC` stateless functional component?
```
const SkeletonRenderer = ({ skeleton, render }) => 
       <SkeletonContext.Consumer>
         {show => (show ? skeleton() : render())}
       </SkeletonContext.Consumer>

SkeletonRenderer.propTypes = {
   skeleton: PropTypes.func.isRequired,
   render: PropTypes.func.isRequired
}
```

Reply:
Also i don't see `contextType` used else where, is this is a library/framework requirement?

Reply:
contextType property is such a requirement. https://reactjs.org/docs/context.html#classcontexttype

Reply:
I'll try and see if making this stateless will work.